app/models/book.rb

```diff
@@ -1,4 +1,12 @@
 class Book < ApplicationRecord
   has_many :rents, dependent: :destroy
-  validates :gender, :author, :image, :title, :editor, :year, presence: true
+  validates :genre, :author, :image, :title, :editor, :year, presence: true
+
+  def current_rent
+    Rent.where(book_id: id).current.first
```

Comment:
Esto debería ser un has_one con scope:

Si no me equivoco es algo similar a:

`has_one :current_rent, -> { where('? BETWEEN start_date AND end_date', Date.current) }, class_name: 'Rent'`

Reply:
@holywyvern  Si, seria una muy buena solución, en ese caso estaría bien manejar el has_many y un has_one con scope o es mejor dejar solo el has_one ?